pkg/scheduler/partition.go

```diff
@@ -380,9 +379,6 @@ func (pc *PartitionContext) removeApplication(appID string) []*objects.Allocatio
 			}
 		}
 	}
-	log.Logger().Debug("application removed from the scheduler",
-		zap.String("queue", queueName),
-		zap.String("applicationID", appID))
```

Comment:
I think this log message can be useful. Right now we have no logs when we remove an application

Reply:
We have more logs now, as part of the change we log this in the context at an INFO level to make sure we log all possible failure cases [moved entry](https://github.com/apache/incubator-yunikorn-core/pull/251/files#diff-a6bde2f6900fb05707fa68db3365f08a79642f39ea629ad5b489d05d3cacc7b3R497-R499)